cypress/e2e/authorization.cy.ts

```diff
@@ -0,0 +1,28 @@
+describe('Authorization', () => {
+  beforeEach(() => {
+    cy.clearCookies();
+  });
+
+  it('should be redirected to login if not authenticated', () => {
+    cy.visit('/');
+    cy.location('pathname').should('eq', '/login');
+  });
+
+  // NOTE this is in scope of FSK-97
+  it.skip('should be redirected to the families page after successful login', () => {
+    cy.login();
+    cy.location('pathname').should('eq', '/families');
+  });
+
+  it('should be able to go to protected routes after logging in', () => {
+    cy.login();
+    cy.visit('/projects');
+    cy.location('pathname').should('eq', '/projects');
+  });
+
+  it('should not be able to go to login after logging in', () => {
+    cy.login();
```

Comment:
This is DRY. I mean using `cy.login()` in every test. at least u can use that in beforeEach function.

Reply:
well in this case we're doing login each time some tests do not require that so we can't put it in a beforeEach

Reply:
In Authorization what could happen? if read about the best practices in cypress this is not a good way. 
u can have `cy.logout()` for special cases.

Reply:
yes we can but that's if we're talking about e2e tests here in that case from the user point of view it's like logging out is part of the test or something